script/install.sh

```diff
@@ -169,17 +170,19 @@ command_help() {
 Description
 
   Download and install a D compiler.
+  By default the latest release of the DMD compiler is selected.
```

Comment:
I wasn't sure whether it's better to have this text here or besides the examples below.